### ISSUE TITLE
Objective values coming back negative

### ISSUE THREAD

**Issue by JWally** [JWally/jsLPSolver#9]
not sure why, but on some problems, the objective value comes back as a negative number. For instance, this throws up a -1460 as result. Any ideas why?

```javaScript
{
        "name": "Integer Clothing Shop Problem II",
        "optimize": "profit",
        "opType": "max",
        "constraints": {
            "yards": {
                "max": 150
            },
            "hours": {
                "max": 200
            }
        },
        "variables": {
            "coat": {
                "hours": 10,
                "yards": 3,
                "profit": 50,
                "coat": 1
            },
            "pants": {
                "hours": 4,
                "yards": 5,
                "profit": 40,
                "pants": 1
            },
            "hat": {
                "hours": 12,
                "yards": 1,
                "profit": 10,
                "hat": 1
            },
            "socks": {
                "hours": 0.5,
                "yards": 0.5,
                "profit": 0.5,
                "socks": 1
            }
        },
        "ints": {
            "coat": 1,
            "pants": 1,
            "hat": 1,
            "socks": 1
        },
        "expects": {
            "feasible": true,
            "coat": 10,
            "pants": 24,
            "hat": 0,
            "result": 1460
        }
    }
```

**Comment by bchevalier** [comment]
oh yeah, i probably forgot to multiply the value of the objective by -1 in case of a maximization problem.

That should be done in the ´´´compileSolution ´´´ function. I will fix it.

**Comment by bchevalier** [comment]
After looking at the code, it like the objective value should be positive. I will investigate, but it will probably be an easy fix.

**Comment by JWally** [comment]
Thanks for looking into it so quickly.

FYI, to the main "Solve" function, I added a third argument for detail. If omitted or false, it returns the solution like it used to. Otherwise it returns the data as you built it. This lets me use the updated library without breaking stuff I built that depended on it.

Try running 

grunt test 

in a shell (install grunt globally if you want and haven't yet). The command will lint the file and run my little test suite against it. 

Really like the direction you're wanting to go with this.
